tests/Event/ReadModel/Permission/ProjectorTest.php

```diff
@@ -68,7 +68,7 @@ public function it_adds_permission_to_the_user_identified_by_the_createdby_eleme
             $payload
         );
 
-        $userId = new StringLiteral('123');
+        $userId = 'dcd1e123-0608-4824-afe3-99124feda64b';
 
         $this->userIdResolver->expects($this->once())
             ->method('resolveCreatedByToUserId')
```

Comment:
I guess this comes from a mergeConflict, but there is a StringLiteral in the next line, that could be removed.
Same with some other markResourceEditableByUser() in this file

Reply:
Removed the other string literals

Reply:
My mistake, should have been more clear 🫣, they are also in `markResourceEditableByUser()` in `tests/Place/ReadModel/Permission/ProjectorTest.php`

Reply:
Updated it